package.json

```diff
@@ -161,6 +161,7 @@
     "i18next-localstorage-backend": "4.2.0",
     "ky": "0.25.1",
     "lodash": "4.17.21",
+    "lodash.combinations": "^18.11.1",
```

Comment:
I always being picky about new imports. Perhaps, it would be more effective to just have the single `permutations` function implemented locally using the native stuff: https://stackoverflow.com/questions/43241174/javascript-generating-all-combinations-of-elements-in-a-single-array-in-pairs